server/webserver.go

```diff
@@ -1142,11 +1133,12 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 	})
 
 	mux.HandleFunc("/maxGasPrice", func(w http.ResponseWriter, r *http.Request) {
-		b, err := s.LivepeerNode.Eth.Backend()
-		if err != nil {
-			respondWith400(w, err.Error())
+		if s.LivepeerNode.Eth == nil {
```

Comment:
Added this check so `livepeer_cli` doesn't immediately crash the node when running in off-chain mode. Though other things can probably crash the node when running in off-chain mode. I haven't take a thorough look - this just happened to be the one crash scenario that stood out to me.